src/main/java/Components/Utils/Component.java

```diff
@@ -0,0 +1,48 @@
+package Components.Utils;
+
+
+import Components.ComponentExceptionHandler.ComponentHandler;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+
+public class Component extends RemoteWebDriver {
+
+    private static String identifier;
+
+    private static WebElement parent;
+
+    public static <T extends BaseComponent> WebElement getComponent(Class<T> ob, String ident) {
+        setIdentifier(ident);
+        T object = ComponentHandler.catchComponentInstanceEx(ob);
+        return object.build();
+    }
+
+    public static <T extends BaseComponent> T component(Class<T> ob, String ident) {
+        setIdentifier(ident);
+        return ComponentHandler.catchComponentInstanceEx(ob);
+    }
+
+    public static <T extends BaseComponent> WebElement getComponent(Class<T> ob, String ident, WebElement part) {
+        setParent(part);
+        setIdentifier(ident);
```

Comment:
This method body seems similar to `getComponent` method. Is it possible to remove code duplication here?